docs/example_list.py

```diff
@@ -38,28 +38,30 @@ def process_index_rst(path):
     "widgets": "Widgets"
 }
 
+# What about animage?
```

Comment:
What about it?

Reply:
> What about it?

cc:  @kisvegabor 

Wow!  I could SWEAR I answered this Friday, but perhaps I didn't push the final button to post it....  If that's the case, I apologize for the delay.

The answer is:  Good catch!  This was an oversight.  I put the comment in there to remind me to look into "animage" and then I did but forgot to remove the comment.  I did so with the subsequent commit.

Kind regards,
Vic

Reply:
Sorry, I was just slow to approve. Your commit message clearly communicated the answer.